.pre-commit-config.yaml

```diff
@@ -21,7 +21,7 @@ repos:
     rev: 3.8.4
     hooks:
     - id: flake8
-      additional_dependencies: ["flake8-mutable", "flake8-fixme", "flake8-docstrings", "flake8-rst-docstrings", "pygments", "flake8-eradicate", "flake8-debugger"]
+      additional_dependencies: ["flake8-mutable", "flake8-docstrings", "flake8-rst-docstrings", "pygments", "flake8-eradicate", "flake8-debugger"]
```

Comment:
removing fixme accoring to discussion in https://github.com/GridTools/gt4py/pull/181